controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -197,6 +199,16 @@ def test_delete_volume_still_copy_fcmaps_not_removed(self):
         with self.assertRaises(array_errors.ObjectIsStillInUseError):
             self.svc.delete_volume("volume")
 
+    def test_delete_volume_found_hyper_swap(self):
```

Comment:
consider replacing `found` with something like `in`

Reply:
Done